views/plugin/featureaction/featuresoundactionconfig.html

```diff
@@ -0,0 +1,19 @@
+<div class="feature-action-sound-config">
+  <select
+      ng-model="sound"
+      ng-change=" ctrl.onSoundChange()"
+      style="width:100%"
+      ng-options="val for val in sounds">
+  </select>
+  <table>
+    <tr>
+      <td class="control-sound-play">Time Between Plays:</td>
```

Comment:
Change this label to `Delay`, give it a tooltip to explain what it does, and put it on the same row as the control.

For inline label/control, you can wrap the two in a block like this:
```
<div class="d-flex flex-row form-group">
  <label class="form-control-label text-right">My Label</label>
  ... control ...
</div>

Reply:
This change is now current with the last commit